runtime/parser/declaration.go

```diff
@@ -75,6 +75,9 @@ func parseDeclaration(p *parser, docString string) (ast.Declaration, error) {
 
```

Comment:
@dsainati1 Do we want to maybe enforce that the order of the access modifier and the view modifier is access then view, and forbid view then access?

Reply:
Is there a good reason to do so? In the long run if we add more keywords I don’t think we’d want to enforce that they appear in a specific order.

Reply:
Mainly to enforce consistency and improve readability. For example, imagine a case:

```cadence
resource R {
  ...
  
  pub view fun foo() { ... }

  pub fun bar() { ... }

  view pub fun baz() { ... }

  pub fun blub() { ... }

}
```

IMHO it's hard to see the third function is also public. 

It might better to only have one way, so there isn't even a question if order matters.
Then again I don't know if any other language enforces such an order.

Reply:
Also, just noticed that the access modifier position is assumed the start position of the function declaration at the moment

Reply:
If we were to enforce an order I think it might make sense to require the access modifier first, and then any other annotation keywords in any order after that. Iirc Java requires `public` or `private` or such to come first, and then allows modifiers like `static`, `final` or `volatile` to come after.

Reply:
Agreed